recipes-omnect/initrdscripts/omnect-os-initramfs/grub-sh

```diff
@@ -37,31 +37,34 @@ set_bootloader_env_var()
     fi
 }
 
-save_fsck_status_base64()
+save_fsck_status()
 {
     local part_name="${1}"
     local source_file="${2}"
     local fsck_res="${3}"
 
+    gzip -9 "${source_file}"
     if [ ! "${part_name}" = "boot" ]; then
         test_and_mount_boot || return 1
-        base64 -w0 ${source_file} > ${ROOTFS_DIR}/boot/fsck.${part_name}
+        base64 -w0 ${source_file}.gz > ${ROOTFS_DIR}/boot/fsck.${part_name}
     elif [ ! "${fsck_res}" = "2" ]; then
         test_and_mount_boot || return 1
-        set_bootloader_env_var omnect_fsck_boot $(base64 -w0 ${source_file}) || set_bootloader_env_var omnect_fsck_boot $(echo "fsck output to big" | base64 -w0 -)
+        set_bootloader_env_var omnect_fsck_boot $(base64 -w0 ${source_file}.gz) || set_bootloader_env_var omnect_fsck_boot $(echo "fsck output to big" | gzip - | base64 -w0 -)
```

Comment:
imo it would be better to use "gzip -9c ${source_file} | base64 ..." here instead of modifying input file and risking potential trouble in calling function

Reply:
done